### ISSUE TITLE
About the data SST-2 format？

### ISSUE THREAD

**Issue by scnb** [kabirahuja2431/FineTuneBERT#1]
In your code, the data files you use is .tsv, while the online data set Standford NLP website offered is .txt, and when I use these .txt files in my code, erros occured. So I wonder if you could upload the .tsv files to github?

**Comment by kabirahuja2431** [comment]
You can download the files in .tsv format form here: https://gluebenchmark.com/tasks
Let me know if it works for you.

**Comment by scnb** [comment]
Thanks a lot.It works.And I accidently changed one line code in dataloader.py, which is
Originally `self.df = pd.read_csv(filename, delimiter = '\t')`
After changed `self.df = pd.read_csv(filename, delimiter = ' ')`
then the program run failed. During debug time, I found that in .tsv files, sentence and label are seperated by two blank spaces, but doesn't '\t' means 4 blank spaces?
By the way, the article you write is very clear and easy to understand,thanks again.

**Comment by kabirahuja2431** [comment]
Glad you liked the article. A tab can be different number of columns depending on the environment that you are using, so its not fixed to 4. I hope this answers your doubt.

**Comment by kabirahuja2431** [comment]
I hope everything is working fine for you now. I am closing the issue.
